requirements.txt

```diff
@@ -0,0 +1,3 @@
+torch==1.7.1
+clip @ git+https://github.com/openai/CLIP.git
```

Comment:
fix the commit lr tag so that the API is stable

Reply:
done